extensions/wikia/CreateNewWiki/templates/CreateNewWiki_Index.php

```diff
@@ -91,30 +82,21 @@
 			<h2><?= wfMessage( 'cnw-desc-headline' ) ?></h2>
 			<p class="creative"><?= wfMessage( 'cnw-desc-creative' )->escaped() ?></p>
 			<form name="desc-form" class="clearfix">
-				<textarea id="Description" placeholder="<?= wfMessage( 'cnw-desc-placeholder' )->escaped() ?>"></textarea>
-				<ol>
-					<li>
-						<?= wfMessage( 'cnw-desc-tip1' )->escaped() ?>
-						<div class="tip-creative"><?= wfMessage( 'cnw-desc-tip1-creative' )->escaped() ?></div>
-					</li>
-					<li>
-						<?= wfMessage( 'cnw-desc-tip2' )->escaped() ?>
-						<div class="tip-creative"><?= wfMessage( 'cnw-desc-tip2-creative' )->escaped() ?></div>
-					</li>
-				</ol>
-
-		        <div class="checkbox" id="all-ages-div"
-					<?= (empty( $selectedLang ) || $selectedLang === $params['LangAllAgesOpt']) ? '' : 'style="display: none"' ?>>
-					<input type="checkbox" name="all-ages" value="1">
-					<?= $app->renderView(
-						'WikiaStyleGuideTooltipIcon',
-						'index',
-						[
-							'text' => wfMessage( 'cnw-desc-all-ages' )->escaped(),
-							'tooltipIconTitle' => wfMessage( 'cnw-desc-tip-all-ages' )->plain(),
-						]
-					 );
-					?>
+				<div class="desc-block">
+					<textarea id="Description" placeholder="<?= wfMessage( 'cnw-desc-placeholder' )->escaped() ?>"></textarea>
+					<div class="checkbox" id="all-ages-div"
+						<?= (empty( $selectedLang ) || $selectedLang === $params['LangAllAgesOpt']) ? '' : 'style="display: none"' ?>>
```

Comment:
should be handled in WW-367 as this is about checkbox. Highlighted because of wrapping in the div.desc-block